### PR TITLE
[jp] Fixed a small Japanese wording

### PR DESCRIPTION
**Pull request by kemkemG0**
fixed a small Japanese wording

### CODE REVIEW

File: content/ja/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Kubernetesを使うとデプロイしたコンテナのあるべき状態を記
 * **自己修復**
 Kubernetesは、処理が失敗したコンテナを再起動し、コンテナを入れ替え、定義したヘルスチェックに応答しないコンテナを強制終了します。処理の準備ができるまでは、クライアントに通知しません。
 * **機密情報と構成管理**
-Kubernetesは、パスワードやOAuthトークン、SSHキーのよう機密の情報を保持し、管理することができます。機密情報をデプロイし、コンテナイメージを再作成することなくアプリケーションの構成情報を更新することができます。スタック構成の中で機密情報を晒してしまうこともありません。
+Kubernetesは、パスワードやOAuthトークン、SSHキーの要機密の情報を保持し、管理することができます。機密情報をデプロイし、コンテナイメージを再作成することなくアプリケーションの構成情報を更新することができます。スタック構成の中で機密情報を晒してしまうこともありません。
```
Comment (atoato88) [inline content/ja/docs/concepts/overview/what-is-kubernetes.md:72]:
```suggestion
Kubernetesは、パスワードやOAuthトークン、SSHキーなどの機密の情報を保持し、管理することができます。機密情報をデプロイし、コンテナイメージを再作成することなくアプリケーションの構成情報を更新することができます。スタック構成の中で機密情報を晒してしまうこともありません。
```
Isn't this word mapped to `such as` in original `en` content? If so, `SSHキーのような` or `SSHキーなどの` is better for translation in `ja`

Original text is following on [last item on here](https://deploy-preview-40329--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/#why-you-need-kubernetes-and-what-can-it-do):
```
Kubernetes lets you store and manage sensitive information, such as passwords, OAuth tokens, and SSH keys. 
```

**Comment by atoato88** [comment]
I left just one comment, please check this.

File: content/ja/docs/concepts/overview/what-is-kubernetes.md (hunk 1 above)
Comment (kemkemG0) [reply content/ja/docs/concepts/overview/what-is-kubernetes.md:72]: @atoato88 
I completely agree with you, and I did not check the original text of that part. I think your correction is perfect, so I will commit it. Thank you for your review!

**Comment by inductor** [comment]
It definitely came from のような

**Comment by atoato88** [comment]
@kemkemG0 
Could you change with the title of PR which describe the change of current content.
I think current title("要機密") does not describe current content.

**Comment by kemkemG0** [comment]
@atoato88 
I wondered about it too and changed it to an English-only title!

**Comment by atoato88** [comment]
Thank you for quick comment.

This PR needs rebase, could you rebase with `main` branch?

**Comment by atoato88** [comment]
@kemkemG0 
Gentle ping.
Could you rebase on the `main` branch?

**Comment by kemkemG0** [comment]
I noticed that the file was already deleted and moved. So I will close this PR.

**Comment by kemkemG0** [comment]
@atoato88 

I was not able to re-open this PR, and I created a new one here;

https://github.com/kubernetes/website/pull/40476

I'm sorry for that and thank you so much for your support.

**Comment by atoato88** [comment]
No ploblem!
Our goal is doing better work even if we have any extra steps.

Thank you for creating other PR, I'll check it.

**Pull request closed without merging**